pages/post/[slug].js

```diff
@@ -101,7 +120,7 @@ export default function Post() {
                   <Image
                     width={200}
                     height={200}
-                    src={post.image_url || ""}
+                    src={post.image.data.attributes.url || ""}
```

Comment:
Let's change `post.image_url` in the `setPostFields` method in utils.js. So don't have to change everywhere